client/src/components/Navbar.js

```diff
@@ -2,15 +2,51 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Nav, Navbar, NavItem, NavDropdown, MenuItem } from 'react-bootstrap';
 import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { logSignerOut } from '../redux/actions/signature';
 
-const NavBar = props => {
-  let homeText;
-  if (props.location.pathname === '/') {
-    homeText = 'RE:IMAGINE DENVER';
-  } else {
-    homeText = 'HOME';
+// RENDERS MyCampaignNavItem BASED ON AUTH STATUS
```

Comment:
move this comment to where MyCampaignNavItem is created in NavBar.
This comment only hold true for how NavBar is creating MyCampaignNavItem but someone else down the line could create it with different logic not based on auth status.